docs/admin/audit-logs.md

```diff
@@ -13,6 +13,18 @@ This feature tracks **create, update and delete** events for the following resou
 - APIKey
 - User
 
+## Filtering logs
+
+In the Coder UI you can filter your audit logs using the pre-defined filter or by using the Coder's filter query like the examples below:
+
+- `resource_type:workspace action:delete` to find deleted workspaces
+- `resource_type:template action:create` to find created templates
+
+The supported filters are:
+
+- `resource_type` - The type of the resource. It can be a workspace, template, user, etc. You can [find here](https://pkg.go.dev/github.com/coder/coder@main/codersdk#ResourceType) all the resource types that are supported.
+- `action`- The action applied to a resource. You can [find here](https://pkg.go.dev/github.com/coder/coder@main/codersdk#AuditAction) all the actions that are supported.
+
```

Comment:
@bpmct I would appreciate your help with this doc

Reply:
This works well for now! Sorry I missed it